modules/apps/transfer/types/ack.go

```diff
@@ -0,0 +1,27 @@
+package types
+
+import (
+	"fmt"
+
+	channeltypes "github.com/cosmos/ibc-go/v8/modules/core/04-channel/types"
+)
+
+// NewForwardErrorAcknowledgement returns a new error acknowledgement with path forwarding information.
+func NewForwardErrorAcknowledgement(packet channeltypes.Packet, ack channeltypes.Acknowledgement) channeltypes.Acknowledgement {
+	ackErr := fmt.Sprintf("forwarding packet failed on %s/%s: %s", packet.GetSourcePort(), packet.GetSourceChannel(), ack.GetError())
+	return channeltypes.Acknowledgement{
+		Response: &channeltypes.Acknowledgement_Error{
+			Error: ackErr,
+		},
+	}
+}
+
+// NewForwardTimeoutAcknowledgement returns a new error acknowledgement with path forwarding information.
+func NewForwardTimeoutAcknowledgement(packet channeltypes.Packet) channeltypes.Acknowledgement {
+	ackErr := fmt.Sprintf("forward packet timeout: source: %s/%s", packet.GetSourcePort(), packet.GetSourceChannel())
```

Comment:
Just to keep the error string more similar to the other one:

```suggestion
	ackErr := fmt.Sprintf("forwarding packet timed out on %s/%s", packet.GetSourcePort(), packet.GetSourceChannel())
```